src/redux/news/news.types.js

```diff
@@ -0,0 +1,2 @@
+export const SET_NEWS = 'SET_NEWS';
+export const NEWS_LOADED = 'NEWS_LOADED';
```

Comment:
It is better to have two types:
- `GET_NEWS` - invokes saga
- `SET_NEWS` - set data in reducer

Reply:
done